modules/smart-agent_elasticsearch/conf/02-cluster_initializing_shards.yaml

```diff
@@ -0,0 +1,21 @@
+module: ElasticSearch
+name: "cluster initializing shards"
+aggregation: true
```

Comment:
If you have multiple clusters, this will do the transformation on all clusters 😃